src/Sentry/Package.cs

```diff
@@ -57,5 +59,28 @@ public static Package FromJson(JsonElement json)
 
             return new Package(name, version);
         }
+
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                return (Name.GetHashCode() * 397) ^ Version.GetHashCode();
```

Comment:
Why 397?

Reply:
prime number

Reply:
I never actually knew why, that's just auto-generated for me haha. Good to know why though!